website/src/content/help-needed.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Help needed?
+---
+
+<Subtitle>
+
+In this page we will document some things that might help clarify certain topics or issue.
+
+</Subtitle>
+
+<Info>
+
+Something is not working and you need help? Do you have a technical question, or are you simply looking for some advice?
+
+If so, we recommend opening a [GitHub Issue](https://github.com/commercetools/merchant-center-application-kit/issues/new/choose) with any inquire that is of your concern.
+
+</Info>
+
+# Browser support
+
+commercetools officially supports the **latest two versions** all major browsers (Google Chrome, Mozilla Firefox, Apple Safari, Microsoft Edge), unless otherwise noted.
+
+The following browsers are not supported any longer:
+
+- `IE 11` and below
+
+<Warning>
+
+It is recommended to always keep your browser up-to-date for **security** reasons.
+
+</Warning>
+
+
+# Changes to the Custom Application are not reflected immediately
+
+When a Custom Application is [registered](/register-applications/configuring-a-custom-application), the first time a user visits the Custom Application route within the Merchant Center, the proxy router fetches the configuration of the Custom Application to determine where to proxy the request to.
+
+This configuration is cached within the proxy router to reduce as much as possible the latency introduced by validating each incoming request. The cache is automatically updated at a regular interval (the default value is **30 minutes**).
+
+This implies that changes to the Custom Application configuration, for example the **Application URL**, are effectively applied according to the cache interval.
+
+<Info>
+
+In the future, the cache mechanism will be improved to provide instant cache invalidation whenever something changes in the Custom Application settings.
+
+</Info>
+
+# Development login does not work on Chrome
+
+In Chrome there is a setting option that prevents saving cookies, which makes it impossible to have a valid session when developing the Custom Applications locally. The option **must be turned on**.
```

Comment:
```suggestion
Chrome provides setting which prevents saving any cookies, which makes it impossible to have a valid session when developing the Custom Applications locally. The option **must be turned off**.
```

Reply:
FYI: The setting must be on, not off. The setting in question is:

```
Allow sites to save and read cookie data (recommended)
```